pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -314,6 +315,14 @@ public ConsumerBuilderImpl(PulsarClientImpl client, Schema<T> schema) {
         return this;
     }
 
+    @Override
+    public ConsumerBuilder<T> batchReceivePolicy(BatchReceivePolicy batchReceivePolicy) {
+        checkArgument(batchReceivePolicy != null, "batchReceivePolicy must not be null.");
```

Comment:
@codelipenghui we have already moved all the validation to `ConsumerConfigurationData`. Can you move the validation there?

Reply:
I see all the validations are still in ConsumerBuilderImpl.